library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/Section.java

```diff
@@ -39,11 +40,13 @@
      * @param itemResourceId layout resource for its items
      * @param loadingResourceId layout resource for its loading state
      * @param failedResourceId layout resource for its failed state
+     * @param emptyResourceId layout resource for its empty state
      */
-    public Section(int itemResourceId, int loadingResourceId, int failedResourceId) {
```

Comment:
Unfortunately these changes in the constructors will "break" backwards compatibility. We will have to adopt a different strategy here.

Could you please create a separate PR merging into `develop` for the Empty state changes?

Reply:
Hi when I use add empty view in section
public PaymentCardAndAccountSection(Context context, int headerLayoutId) {
        super(new SectionParameters.Builder(R.layout.item_view_payment_account)
                .headerResourceId(headerLayoutId)
                .emptyResourceId(R.layout.payment_account_empty_view)
                .build());
        this.context = context;
    }

there is a error:
Caused by: java.lang.IllegalArgumentException: Stateless section shouldn't have an empty state resource, 
Can you help me, please

Reply:
Change your PaymentCardAndAccountSection to extend Section instead of StatelessSection. Check out [Example3](https://github.com/luizgrp/SectionedRecyclerViewAdapter/blob/c64d766689fa18b87dd202c0bc998cc62a79eb98/app/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/demo/Example3Fragment.java) of the demo app.